examples/with-deta-base/README.md

```diff
@@ -0,0 +1,38 @@
+# Next.js + Deta Base
```

Comment:
Let's update this readme so it follows this structure: https://github.com/vercel/next.js/tree/canary/examples/with-userbase - Also update the deploy button so it requires the `DETA_PROJECT_KEY` when importing the project to Vercel.